lua/lir/config.lua

```diff
@@ -6,7 +6,7 @@ local defaults_values = {
   devicons_enable = false,
   mappings = {},
   float = {
-    size_percentage = 0.5,
+    size_percentage = { width = 0.5, height = 0.5 },
```

Comment:
I had to do this, as `vim.tbl_deep_extend` was giving error otherwise. If there is a workaround, please do tell :)

Reply:
LGTM